docs/apis-tools/frontend-development/01-task-applications/02-user-task-lifecycle.md

```diff
@@ -76,21 +76,31 @@ Our best practices, as implemented in Tasklist, are as follows:
 
 Make sure that you create your own validation logic that matches your use case.
 
-## Implement the life cycle with the task API
+## Implement task life cycle with the Camunda 8 API
 
-To implement task life cycle operations with the task API, call the respective endpoints:
+Use the Camunda 8 REST API to implement task life cycle operations. You can find the API specifications [here](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md).
 
-- [`POST /user-tasks/:taskKey/assignment`](/apis-tools/zeebe-api-rest/specifications/assign-a-user-task.api.mdx) or [`DELETE /user-tasks/:taskKey/assignee`](/apis-tools/zeebe-api-rest/specifications/unassign-a-user-task.api.mdx) to change task assignment.
-- [`PATCH /user-tasks/:taskKey`](/apis-tools/zeebe-api-rest/specifications/update-a-user-task.api.mdx) to update a task.
-- [`POST /user-tasks/:taskKey/completion`](/apis-tools/zeebe-api-rest/specifications/complete-a-user-task.api.mdx) to complete a task.
+Task life cycle events can be tracked via the following API endpoints:
 
-All these endpoints (except `DELETE`) allow you to send a custom `action` attribute via the payload. The `action` attribute carries any arbitrary string and can be used to track any life cycle event, including those mentioned above.
+- Assign user task:
+  - [`POST /user-tasks/:userTaskKey/assignment`](/apis-tools/camunda-api-rest/specifications/assign-user-task.api.mdx)
+  - [`DELETE /user-tasks/:userTaskKey/assignee`](/apis-tools/camunda-api-rest/specifications/unassign-user-task.api.mdx) is used to unassing a user task.
+- Update user task:
+  - [`PATCH /user-tasks/:taskKey`](/apis-tools/camunda-api-rest/specifications/update-user-task.api.mdx)
+- Complete user task:
+  - [`POST /user-tasks/:taskKey/completion`](/apis-tools/camunda-api-rest/specifications/complete-user-task.api.mdx)
 
-#### [`POST /user-tasks/:taskKey/assignment`](/apis-tools/zeebe-api-rest/specifications/assign-a-user-task.api.mdx)
+The endpoints above (except `DELETE`) allow you to send a custom `action` attribute via the payload. The `action` attribute carries any arbitrary string and can be used to track any life cycle event, including those mentioned above.
```

Comment:
Non-blocking suggestion: Should we say above twice? If the content is moved, this reference then becomes wrong/orphaned - it is always useful where possible to not directly refer to the physical location of content, e.g. you could say instead:

Task life cycle events can be tracked via the following API endpoints.

You can use these endpoints (except `DELETE`) to send a custom `action` attribute via the payload. The `action` attribute carries any arbitrary string and can be used to track any life cycle event.